locallibs/userpkg.py

```diff
@@ -88,6 +88,9 @@
     /usr/bin/killall DirectoryService 2>/dev/null || /usr/bin/killall opendirectoryd 2>/dev/null
 """
 
+PI_CREATEHOMEDIR = """
+    /usr/sbin/createhomedir -c
+"""
```

Comment:
I don't think you've addressed my issue with `createhomedir` not working for anything other than installs to /.
I think the postinstall script bit needs to test $3 and do the right thing, and I think the overall `createuserpkg` tool needs to warn people in the help output that this option works only in some scenarios; etc.

Reply:
I will add the createhomedir command to the pi_live_actions, so it gets written into the if [ "$3" == "/" ] block. Additionally i will add a notice to the help text that says that it only work an a boot volume.
Would that be ok?
Thanks again
Gerd